tokio/src/macros/select.rs

```diff
@@ -396,8 +396,27 @@
 ///     }
 /// }
 /// ```
+#[cfg(doc)]
 #[macro_export]
 #[cfg_attr(docsrs, doc(cfg(feature = "macros")))]
+macro_rules! select {
+    {
+        $(
+            biased;
+        )?
+        $(
+            $bind:pat = $fut:expr $(, if $cond:expr)? => $handler:expr /*optional comma*/,
```

Comment:
I might even drop the comment about the comma. Doesn't seem that important.
```suggestion
            $bind:pat = $fut:expr $(, if $cond:expr)? => $handler:expr,
```

Reply:
👍